src/models/BibDetails.ts

```diff
@@ -3,18 +3,39 @@ import type {
   LinkedBibDetail,
   BibDetail,
   FieldMapping,
+  AnnotatedMarcField,
+  Url,
+  SubjectHeadingDetail,
+  AnnotatedMarc,
+  AnyBibDetail,
 } from "../types/bibDetailsTypes"
 
-interface BibDetails extends Bib {
-  groupedNotes: object
-}
 export default class BibDetailsModel {
```

Comment:
I know this went in previously, but could you change the name of the class to BibDetails to follow the naming convention of the other models?